app/routes/controllers/roverImage.controller.js

```diff
@@ -0,0 +1,24 @@
+﻿const Exception = require("../../customExceptions/Exception");
+const express = require("express");
+const {getMostRecentRoverImage} = require("../../services/roverImage.service");
+const {nasaApiUrl} = require('../../config/config');
+
+const router = express.Router();
+
+router.post('/rover-image', async (req, res, next) => {
+    try {
+        const {apiKey} = req.body;
+
+        if (!apiKey) {
```

Comment:
also u can add check on !== string
`
if (!apiKey || typeof apiKey !== 'string') {
            return res.status(400).json({ message: 'Valid API key is required' });
        }
`